Dockerfile

```diff
@@ -13,7 +13,6 @@ ARG NODE_VERSION=lts
 ARG WATCHMAN_VERSION=4.9.0
 
 # set default environment variables
-ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
Is `openjdk:8-slim` has `JAVA_HOME` env. It's needed by react-native.

Reply:
yep, CI is green and please check https://github.com/docker-library/openjdk/blob/7a33416016b60c045cf0ba99e82617ed6c130595/8/jdk/Dockerfile

Reply:
Actually it's checked by script in react-native repo. Maybe we deal with it later.